manifests/registry.pp

```diff
@@ -48,9 +48,9 @@
   $docker_command = $docker::params::docker_command
 
   if $::osfamily == 'windows' {
-    $exec_environment = ['PATH=C:/Program Files/Docker/']
+    $exec_environment = ["PATH=${::program_files_path}/Docker/"]
```

Comment:
Setting PATH and `exec_path` is bad practice

https://puppet.com/docs/puppet/5.3/types/exec.html#exec-attribute-environment